src/Composer/Satis/Builder/ArchiveBuilder.php

```diff
@@ -99,9 +101,19 @@ public function dump(array $packages)
                 if (!$this->skipErrors) {
                     throw $exception;
                 }
+
+                //If file exists then exclude it
```

Comment:
@alcohol Because if you do not add the name of a file into `$excludeFiles`, it will be removed